app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Climate Watch NDC Tracking tool</title>
+    <title>Country Platform on South Africa’s Biennial Update Report on Climate Change</title>
     <%= csrf_meta_tags %>
-
+     
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <% if @is_production %>
       <%= stylesheet_pack_tag 'main' %>
     <% end %>
+    
+    <link rel="shortcut icon" href="http://www.iconj.com/ico/2/7/2740u33ncy.ico" type="image/x-icon" />
```

Comment:
I love the icon!

But we are missing some icons for retina and other devices so could you please generate all the favicons with a tool like this: https://realfavicongenerator.net/ and include them?

Thanks!

Reply:
uaaa, what a great tool! thank you